CONTRIBUTING.md

```diff
@@ -0,0 +1,109 @@
+# Contributing to Open Foris Arena
+
+## Prerequisites
+
+First, [install Yarn](https://yarnpkg.com/en/docs/install) (a modern npm replacement).
+
+Then, install [Node.js](https://nodejs.org/en/download/) (currently we are using LTS version 18.x).
+
+### GitHub packages: authentication
+
+Arena requires some GitHub packages (@openforis/arena-core and @openforis/arena-server).
+In order to do that, you must use a token with (at least) the `read:packages` scope.
+
+You can authenticate to GitHub Packages with npm by either editing your per-user ~/.npmrc file to include your personal access token or by logging in to npm on the command line using your username and personal access token.
+
+To authenticate by adding your personal access token to your ~/.npmrc file, edit the ~/.npmrc file for your project to include the following line, replacing TOKEN with your personal access token. Create a new ~/.npmrc file if one doesn't exist and replace TOKEN with your personal access token.
+
+```shell
+//npm.pkg.github.com/:_authToken=TOKEN
+```
+
+To authenticate by logging in to npm, use the npm login command, replacing USERNAME with your GitHub username, TOKEN with your personal access token, and PUBLIC-EMAIL-ADDRESS with your email address.
+
+If GitHub Packages is not your default package registry for using npm and you want to use the npm audit command, we recommend you use the --scope flag with the owner of the package when you authenticate to GitHub Packages.
+
+```shell
+$ npm login --scope=@OWNER --registry=https://npm.pkg.github.com
+
+> Username: USERNAME
+> Password: TOKEN
+> Email: PUBLIC-EMAIL-ADDRESS
+```
+
+> For more information see [Authenticating to GitHub Packages](https://docs.github.com/en/packages/guides/configuring-npm-for-use-with-github-packages#authenticating-to-github-packages)
+
+## Development
+
+To install local database:
+
+```shell script
+sudo docker run -d --name arena-db -p 5444:5432 -e POSTGRES_DB=arena -e POSTGRES_PASSWORD=arena -e POSTGRES_USER=arena postgis/postgis:12-3.0
+```
+
+In order to access Arena when it's installed on a remote server, you need to request access to it or being invited to it.
+To simplify this process when working locally, you can insert a test user with System Administrator rights (username: test@openforis-arena.org - password: test) directly into the database running this SQL script with the SQL client you prefer:
+
+```
+INSERT INTO "user" (name, email, PASSWORD, status)
+VALUES ('Tester', 'test@openforis-arena.org', '$2a$10$6y2oUZVrQ7aXed.47h4sHeJA8VVA2dW9ObtO/XLveXSzQKBvTOyou', 'ACCEPTED');
```

Comment:
I think that this is only valid if you use the same secret

Reply:
test user can be added at the first startup using the ADMIN_USER and ADMIN_PASSWORD environment variables, I will modify this part of the documentation.